example/volume_mount.nomad

```diff
@@ -0,0 +1,34 @@
+job "volume_mount" {
+  datacenters = ["dc1"]
+
+  group "volume_mount-group" {
+
+    volume "data" {
+      type = "host"
+      source = "s1"
+      read_only = false
+    }
+
+    task "volume_mount-task" {
+      driver = "containerd-driver"
+      config {
+        image           = "docker.io/library/ubuntu:16.04"
+        command         = "sleep"
+        args            = ["600s"]
+      }
+
+      volume_mount {
+        destination = "/tmp/t1"
+        volume = "data"
+      }
+
+      resources {
+        cpu    = 500
+        memory = 256
+        network {
+          mbits = 10
```

Comment:
This setting is deprecated, if you choose to leave it that is OK as well. Just an FYI if you want to remove it or not. https://www.nomadproject.io/docs/job-specification/network#mbits

Reply:
@robloxrob Aah! Let's leave this change out of the scope of this PR, since this would have to be removed from all the [`example`](https://github.com/Roblox/nomad-driver-containerd/tree/master/example) jobs.
I will open a separate PR for this change. Tracking issue [`here`](https://github.com/Roblox/nomad-driver-containerd/issues/53)

Reply:
Agreed. Thank you!

Reply:
I just remove these lines for we do not care the network here.